assets/js/src/form_editor/components/form_settings/form_placement_options/below_pages.tsx

```diff
@@ -17,11 +20,27 @@ const BelowPages = () => {
   );
 
   return (
-    <FormPlacementOption
-      label={MailPoet.I18n.t('placeFormBellowPages')}
-      icon={Icon}
-      active={placeFormBellowAllPages || placeFormBellowAllPosts}
-    />
+    <>
+      <FormPlacementOption
+        label={MailPoet.I18n.t('placeFormBellowPages')}
+        icon={Icon}
+        active={placeFormBellowAllPages || placeFormBellowAllPosts}
+        onClick={() => setDisplaySettings(true)}
```

Comment:
It seems that modal open/close logic is spread across two components: `FormPlacementOption` and specific `BelowPages` components. I can see in spec. that all form placement component will have modal with settings so what about motiving modal and it's open/close logic to common component and pass only modal content as children here?